modules/gltf/structures/gltf_node.cpp

```diff
@@ -176,3 +179,15 @@ GLTFLightIndex GLTFNode::get_light() {
 void GLTFNode::set_light(GLTFLightIndex p_light) {
 	light = p_light;
 }
+
+Variant GLTFNode::get_additional_data(const String &p_extension_name) {
+	return additional_data[p_extension_name];
```

Comment:
What happens if additional_data does not container extension name?

Reply:
It returns null, which actually makes me realize, we don't need `has_additional_data`. I've documented this behavior and removed `has_additional_data`.